assets/src/components/ui/menu/step-bubble/style.css

```diff
@@ -0,0 +1,74 @@
+.ee-step-bubble-menu-container {
+    width: 100%;
+    position: relative;
+    margin: auto;
+}
+
+.ee-step-bubble-menu-container .step-bubbles-container {
+    display: inline-block;
+    position:relative;
+    margin: auto;
+}
+
+.step-bubbles-container .progress-step-line {
+    background: #ddd;
+    border: 1px solid #ddd;
+    height: 0px;
+    position: absolute;
+    top: 30%;
+    left: -2%;
+    width: 104%;
+    z-index: -1;
+}
+
+.step-bubbles-container .ee-step-bubble-item {
+    display: inline-block;
+    margin-right: 30px;
+    text-align: center;
+    border: none;
+    background-color: transparent;
+}
+
+.step-bubbles-container .ee-step-bubble-item:focus {
+    outline: unset;
+    border: none;
+}
+
+.step-bubbles-container .ee-clickable {
```

Comment:
there is an existing `clickable` class in WP core but that likely only applies to the admin, so this might still be good, but should eventually be moved into a global ee stylesheet (which you are probably already planning to do)

Reply:
Ya just keeping styles in here until the new style system is put in place.